manifest/manifest.go

```diff
@@ -10,19 +10,31 @@ import (
 
 // FIXME: Should we just use docker/distribution and docker/docker implementations directly?
 
+// FIXME(runcom, mitr): should we havea mediatype pkg??
```

Comment:
I don’t quite feel we need to have a separate package for everything. Sure, use packages to simplify dependencies and avoid loops and have clear separation between transports, conversions and signatures, and from time to time to simplify the callers’ notation to avoid too many words in a symbol reference, but at some point adding a complex difficult-to-navigate package structure could make understanding more difficult instead of simpler.